index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Md. Porag Sarkar: Portfolio</title>
+</head>
+
+<body>
+    <div class="container-parent">
+
+        <header class="top-navigation">
+            <div class="container-logo">
+                <a href="#" class="logo">My Logo</a>
+            </div>
+            <nav>
+                <img class="menu-icon" src="./assets/mobile/Icon - Menu.svg" alt="Menu Button">
+            </nav>
+        </header>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job making all the changes mentioned by the previous reviewer. How I think it would be better if you update your header styling by making your `Hamburger` button be visible and the remove the underline decoration styling on the logo text and even increase its fonts(size & weight). 
**I recommend you make these changes before merging your pull request as they were not mentioned by the previous reviewer**.
![Screenshot from 2023-03-24 20-42-42](https://user-images.githubusercontent.com/83042742/227669487-d0531717-7f31-48e1-989d-538ed0c8b9c3.png)

Reply:
Thanks for the recommendation. Will have it done definitely.